.github/workflows/build.yml

```diff
@@ -17,15 +17,16 @@ jobs:
     strategy:
       matrix:
         scala:
-          - { name: "Scala 2", version: "2.12.19", binary-version: "2.12", java-version: "adopt@1.11" }
-          - { name: "Scala 2", version: "2.13.10", binary-version: "2.13", java-version: "adopt@1.11" }
-          - { name: "Scala 3", version: "3.1.3",   binary-version: "3",    java-version: "adopt@1.11" }
+          - { name: "Scala 2", version: "2.12.19", binary-version: "2.12", java-version: "11", java-distribution: "temurin" }
+          - { name: "Scala 2", version: "2.13.10", binary-version: "2.13", java-version: "11", java-distribution: "temurin" }
+          - { name: "Scala 3", version: "3.1.3",   binary-version: "3",    java-version: "11", java-distribution: "temurin" }
 
     steps:
       - uses: actions/checkout@v4
-      - uses: olafurpg/setup-scala@v14
+      - uses: actions/setup-java@v4
         with:
           java-version: ${{ matrix.scala.java-version }}
+          distribution: ${{ matrix.scala.java-distribution }}
```

Comment:
Could you add `cache: sbt` as well here to replace caching?

Reply:
Oh, that's great! I didn't know about that. Thank you!
I'll create another PR to replace `actions/cache@v4` with that.